src/main/java/android/components/CartProductItem.java

```diff
@@ -0,0 +1,31 @@
+package android.components;
+
+import com.zebrunner.carina.utils.factory.DeviceType;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import common.components.CartProductItemBase;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = CartProductItemBase.class)
+public class CartProductItem extends CartProductItemBase {
+
+    @FindBy(xpath = "//android.widget.TextView[@text='%s']/../following-sibling::android.view.ViewGroup[@content-desc='test-Price']/android.widget.TextView")
+    private ExtendedWebElement price;
```

Comment:
always specify type of elements, it will help later during support
```suggestion
    private ExtendedWebElement priceLabel;
```

Reply:
ok